README.md

```diff
@@ -3,8 +3,8 @@ CategoricalArrays.jl
 
 [![Build Status](https://travis-ci.org/JuliaData/CategoricalArrays.jl.svg?branch=master)](https://travis-ci.org/JuliaData/CategoricalArrays.jl)
 [![Build Status](https://ci.appveyor.com/api/projects/status/jq64i3656pwi18pg?svg=true)](https://ci.appveyor.com/project/nalimilan/categoricalarrays-jl)
-[![Julia 0.5 Status](http://pkg.julialang.org/badges/CategoricalArrays_0.5.svg)](http://pkg.julialang.org/?pkg=CategoricalArrays&ver=0.5)
-[![Julia 0.6 Status](http://pkg.julialang.org/badges/CategoricalArrays_0.6.svg)](http://pkg.julialang.org/?pkg=CategoricalArrays&ver=0.6)
+[![Julia 0.7 Status](http://pkg.julialang.org/badges/CategoricalArrays_0.7.svg)](http://pkg.julialang.org/?pkg=CategoricalArrays&ver=0.7)
+[![Julia 1.0 Status](http://pkg.julialang.org/badges/CategoricalArrays_1.0.svg)](http://pkg.julialang.org/?pkg=CategoricalArrays&ver=1.0)
```

Comment:
1.0 badges don't yet exist, and 0.7 badges are woefully outdated and are not updated

Reply:
Yeah but I figure they will be fixed at some point. Is that the case? :-)

Reply:
For sure, though it remains to be seen which comes first: the imminent heat death of the universe or 1.0 testing badges.